docs/usage.md

```diff
@@ -327,9 +327,9 @@ The above pipeline run specified with a params file in yaml format:
 nextflow run genomic-medicine-sweden/tomte -profile docker -params-file params.yaml
 ```
 
-with `params.yaml` containing:
+with:
 
-```yaml
+```yaml title="params.yaml"
```

Comment:
How do you see the name of params.yaml?

I think the old version is clearer.

Reply:
To be honest, I think it looks rather nice this way. It's also clear because line 324 says that it is a params file in yaml, then you see the command with it and after that how it would look, so I will leave it as it is